src/Rector/Dynamic/NamespaceReplacerRector.php

```diff
@@ -96,7 +96,7 @@ private function resolveNameFromNode(Node $node): string
         }
 
         if ($node instanceof Name) {
-            /** @var FullyQualified|null $resolveName */
+            /** @var FullyQualified $resolveName */
             $resolveName = $node->getAttribute(Attribute::RESOLVED_NAME);
```

Comment:
What if the value was not resolved for some special case?
It happened before and broke Rector